CONTRIBUTING.md

```diff
@@ -1,35 +1,44 @@
 # Contributing
 
-Everyone is welcome to contribute to containers. Reach out to team members if you have questions:
+## Requirements
 
-- Matrix chat: [#containers:mozilla.org](https://matrix.to/#/#containers:mozilla.org)
-- Email: containers@mozilla.com
+* Firefox 91.1.0+
+* Git 2.13+
+* Node 7+
 
-## Filing bugs
+## Getting Started
 
-If you find a bug with containers, please file a issue.
+1. Follow the instructions on [How to fork a repository][fork]
+2. Fetch the locales:
 
-Check first if the bug might already exist: https://github.com/mozilla/multi-account-containers/issues
+    ```
+    cd multi-account-containers
+    git submobule update --init
+    ```
+3. Install the project dependencies
+    ```
+    npm install -legacy-peer-deps
+    ```	
+3. Install [web-ext][web-ext] command-line tool:
 
-[Open an issue](https://github.com/mozilla/multi-account-containers/issues/new)
+    ```
+    npm install -g web-ext
+    ```
 
-1. Visit about:support
-2. Click "Copy raw data to clipboard" and paste into the bug. Alternatively copy the following sections into the issue:
-  - Application Basics
-  - Nightly Features (if you are in nightly)
-  - Extensions
-  - Experimental Features
-3. Include clear steps to reproduce the issue you have experienced.
-4. Include screenshots if possible.
+4. Run `web-ext run -s src/`.
```

Comment:
See above comment

```suggestion
4. Run `npm run web-ext`.
```

Reply:
Actually, I was wondering if we shouldn't simply drop a `web-ext-config.js` file in the root folder so `web-ext` will work in both the root folder and `/src`.